distribution/branding-api/src/main/java/org/codice/ddf/branding/BrandingPlugin.java

```diff
@@ -10,20 +10,31 @@
  * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
  * is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
- */
+ **/
+
 package org.codice.ddf.branding;
 
 import java.io.IOException;
 
-/**
- * This service looks up resource paths
- * in bundle fragments attached to
- * {@code org.apache.karaf.webconsole.console}
- */
-public interface BrandingResourceProvider {
-
-    /**
-     * Loads a binary resource (e.g. image) as bytes
-     */
-    public byte[] getResourceAsBytes(String path) throws IOException;
+public interface BrandingPlugin {
+
+    String getProductName();
+
+    String getProductURL();
+
+    String getProductImage();
+
+    String getProductImageAsBytes() throws IOException;
```

Comment:
The AsBytes with a return type of String seems odd.  Maybe `getBase64ProductImage` instead?